docs/references/resources.reference.md

```diff
@@ -21,6 +21,7 @@ Here's list of interesting things to read, watch and listen to if you're working
 * [Web Dev Server](https://modern-web.dev/docs/dev-server/overview/)
 * [Web Test Runner](https://modern-web.dev/docs/test-runner/overview/)
 * [Writing one-sentence-per-line](https://hieuphay.com/one-sentence-per-line/)
+* [Defensive CSS](https://defensivecss.dev/)
```

Comment:
It seems we order these alphabetically so it might be better to move it before "Design tokens ..." :wink: